extensions/sql-migration/src/api/azure.ts

```diff
@@ -654,17 +655,18 @@ export async function validateIrSqlDatabaseMigrationSettings(
 			userName: migration._sqlServerUsername,
 			password: migration._sqlServerPassword,
 			authentication: migration._authenticationType,
+			encryptConnection: encryptConnection,
```

Comment:
for `validateIR` for DB, verify that both `encryptConnection` and `trustServerCertificate` are explicitly passed